gooddata-sdk/gooddata_sdk/catalog/workspace/service.py

```diff
@@ -24,6 +26,59 @@
 ValidObjectsInputType = Union[ValidObjectTypes, List[ValidObjectTypes], ExecutionDefinition]
 
 
+class CatalogWorkspaceService:
+    def __init__(self, api_client: GoodDataApiClient) -> None:
+        self._client = api_client
+        self._entities_api = metadata_apis.EntitiesApi(api_client.metadata_client)
+
+    def list_workspaces(self) -> List[CatalogWorkspace]:
+        get_workspaces = functools.partial(
+            self._entities_api.get_all_entities_workspaces,
+            include=["workspaces"],
```

Comment:
Is it necessary to list workspaces with `include`? I do not see any usage of it. Do I overlook something?

Reply:
When there include is not presented then the call does not return information about parent.

Reply:
Thanks for clarification.